.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: java
+
+stages:
+  - test
+  - name: release
+    if: env(RELEASE) == true AND fork == false
+
+jobs:
+  include:
+    - stage: test
+      script: ./gradlew build
+    - stage: release
```

Comment:
Since we're enabling releases here, we should use `gradle-release:0.5.0` to get the proper POM SCM